crates/db_views/src/post/post_view.rs

```diff
@@ -1263,33 +1293,128 @@ mod tests {
   async fn creator_info(data: &mut Data) -> LemmyResult<()> {
     let pool = &data.pool();
     let pool = &mut pool.into();
+    let community_id = data.community.id;
 
-    // Make one of the inserted persons a moderator
-    let person_id = data.local_user_view.person.id;
-    let community_id = data.inserted_community.id;
-    let form = CommunityModeratorForm {
+    let tegan_listings = PostQuery {
```

Comment:
The `can_mod` tests are below. I added comments to make it clear the interactions between the 3 users (1 admin, 1 higher mod, 1 lower mod).